app/controllers/curation_concerns/permissions_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CurationConcerns::PermissionsController < ApplicationController
+  include CurationConcerns::PermissionsControllerBehavior
+
+  def confirm_access
+  end
```

Comment:
@cam156 I don't know anything about the review workflow yet. I suspect I'll learn about it next week.